### ISSUE TITLE
Remove ./bin and ./utils

### ISSUE THREAD

**Issue by axmmisaka** [lf-lang/lingua-franca#1992]
This is from #1988 

I am thinking removing all of them. Developers should know after building where the artifacts are, and handle PATH themselves should they wish to add these artifacts to their PATH. They should also run `./gradlew :cli:lfc:assemble` at their pleasure. If they are not familiar with how to do so, using `nightly` would be a better idea.

After #1988 there would be a ton of stuff in `./bin`. Yet (I think) these utility scropts doesn't really belong in this repo.

I have this feeling as some projects I've seen (like NSA's Ghidra) do this. Most projects don't have a script dedicated to help people execute the build process, rather they simply point developers to the right direction in README. I think we should do that, but that's my €0.02.

**Comment by axmmisaka** [comment]
One more reason:

> PS: These scripts are kind of a PITA here; bash and pwsh are cross platform, but the scripts are assuming platform based on what people use; I say it's gradle's fault
> `lfc-dev` won't work on `MSysGit`/git bash, and `lfc-dev.ps1` won't work on linux `pwsh`.

**Comment by cmnrd** [comment]
`utils` already got removed by https://github.com/lf-lang/lingua-franca/pull/1995.

Personally, I find the `bin` scripts extremely useful. Not because I don't want to use gradle, but because I can run the script from anywhere in the filesystem (unlike gradle).

> After https://github.com/lf-lang/lingua-franca/pull/1988 there would be a ton of stuff in ./bin. Yet (I think) these utility scropts doesn't really belong in this repo.

What do you mean by a "ton of stuff"?

**Comment by lhstrh** [comment]
I also find the scripts useful and would like to keep them. If we want to make it abundantly clear that this is just meant as dev tooliing, we could rename `bin` to `.bin`...

**Comment by edwardalee** [comment]
For the record, I really prefer to avoid hidden directories.

**Comment by cmnrd** [comment]
> For the record, I really prefer to avoid hidden directories.

Me too. If we rename it, it should probably be to `dev`. But at the moment I don't see a compelling reason for doing so.

**Comment by lhstrh** [comment]
OK, then let's resolve this as a wontfix.
